### DISCUSSION TITLE
Being able to change display text with commands

### DISCUSSION THREAD

**Discussion by G4PLS** [streamdeck-linux-gui/streamdeck-linux-gui#132] [Ideas / Feature Requests]
This Idea sparked by a very specific problem I had.

### For context this was my idea:
I recently wanted to add some media controls to my streamdeck, and that works perfectly
But I wanted to update the Display to show the Volume of a Sink

### Now the Idea generalized:
So I just thought that it would be awesome if we are able to change things by updating the json or something in that direction.
Currently it would require a restart for the changes in the json to take effect (Which makes sense with the current implementation)

**Comment by Danie10** [comment]
Is this not identical to https://github.com/streamdeck-linux-gui/streamdeck-linux-gui/discussions/5 which also updates display?

**Reply by G4PLS** [reply]
Oh yeah they do sound very similar, and now that i read the title again its rather self explanitory
When I read it by looking at all the discussions this didnt even cross my mind

Even though my description differs a little bit, I want to make it possible to control the state of the Button with commands from the cli and not with the button itself, its probably a small difference but this makes it still very different.
At least in my mind atm

Sorry!
